multiphate/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: multiphate 
+  version: 0.5 
+
+source:
+  git_url: https://github.com/carolzhou/multiPhATEb.git
+  sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+
+build:
+  noarch: python 
```

Comment:
you could also add `script: "{{ PYTHON }} -m pip install . --no-deps -vv"` and remove the build.sh script ...

Reply:
Do you mean, replace the "build:  noarch: python" with what you have written there? Should I do this and resubmit the PR?

Reply:
Have a look at this one for example: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/hicexplorer/meta.yaml

You can just get rid of the additional build.sh file when your installation command is really simple.

Reply:
Thank you. That's a very informative example.

Reply:
But why haven't you used it?